script/Arcueid/Day 4/04_00_ARC04_1_1.txt

```diff
@@ -1,13 +1,13 @@
 [sha:cc509395dec627bbbd6ed7e85c207214be94bbec]{
 -- Page 10003, Offset 5731.
 -- 「そういえば。思い出したんだけど、秋葉は今も浅女の生徒なんだよな……？」
-"Oh, by the way.
- If I remember correctly, you still attend Asagami Private Girl's Academy, don't you...?"
+"Oh, that's right――
+if I remember correctly, you still attend Asagami Private Girl's Academy, don't you?"
```

Comment:
feel like just saying "Asagami" here is totally fine.
You don't say the full names of school generally speaking